webvtt.go

```diff
@@ -59,11 +59,15 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	var item = &Item{}
 	var blockName string
 	var comments []string
+	var indexContent []int
+
 	for scanner.Scan() {
 		// Fetch line
 		line = strings.TrimSpace(scanner.Text())
+
+		// Fetch Index
```

Comment:
Can you remove this comment and the empty line above ?

Reply:
fixed with https://github.com/asticode/go-astisub/pull/22/commits/489f53adc2376337082e86637e00f39a74a929b2